### ISSUE TITLE
Doubt about distribution

### ISSUE THREAD

**Issue by antoniogamiz** [RayTracing/raytracing.github.io#700]
In the first book, `RayTracingInOneWeekend`, section `8.5. True Lambertian Reflection`, it says that 

> the distribution scales by `cos^3(phi)`

Anyone knows where's that proven? Could you please send me a reference to some book or similar where that is explained? I mean the mathematics behind that fact.

**Comment by rjkilpatrick** [comment]
This appears to be a duplicate of #530 , I will try and answer this later this week.

**Comment by trevordblack** [comment]
See comment https://github.com/RayTracing/raytracing.github.io/issues/530#issuecomment-660700792

I very much doubt that we can explain (succinctly) that the distribution is `cos^3(phi)`.
I can see if there is (at most) 2 sentences that can lend an intuition.
It's already a pretty over-stuffed chapter and in need of some editing. See #696 #530 #617

Ultimately, I think the best thing would just be a citation, which is blocked by #299 #407

In the meantime, the comment above has what you're looking for.
